plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/AccessTrackingFileSystem.java

```diff
@@ -61,20 +98,209 @@ public FSDataInputStream open(Path f)
         return fileSystemDelegate.open(f);
     }
 
+    @Override
+    public FSDataInputStream open(PathHandle fd)
+            throws IOException
+    {
+        return fileSystemDelegate.open(fd);
```

Comment:
`incrementOpenCount`?

Reply:
`PathHandle` is an interface been implemented by below classes and it doesn't expose the file path. 
- LocalFileSystemPathHandle: `getPath()` exists
- HdfsPathHandle: `getPath()` exists
- RawPathHandle: doesn't expose a path

@findepi Do you want to get the name based on the class?

Reply:
let's throw UnsupportedOperationException then

(let's not spend time on things we currenrtly don't need)